src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -40,10 +40,12 @@ object LocalBackend {
  * Handles both local Docker runs as well as local direct command line executions.
  */
 class LocalBackend extends Backend with LazyLogging {
+
   /**
    * Executes the specified command line, using the supplied lookup function for expression evaluation.
    * Returns a `Map[String, Try[WdlValue]]` of output names to values.
    */
+  // FIXME: During testing I found that "call: Call"'s Command is ignored. Is this correct?
```

Comment:
It ultimately comes from here in `Call.scala`:

      def instantiateCommandLine(inputs: CallInputs): Try[String] = task.command.instantiate(inputs)

The `Try` aspect is used to determine if we ever get to `executeCommand` however, so it gets a bit weird I guess? 

At the very least we should create a tech debt ticket to harmonize this to a single value (in the `Call` or let's pass a `Call` derived object instead)